xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_targets.cpp

```diff
@@ -1,22 +1,96 @@
 #include "stdafx.h"
 #include "rasterizer_dx9_targets.h"
-#include "rasterizer/dx9/rasterizer_dx9_main.h"
+
+#include "rasterizer_dx9_main.h"
+
 #include "camera/camera.h"
+#include "main/main_screenshot.h"
+#include "rasterizer/rasterizer_globals.h"
+
+/* constants */
+
+// This is no longer used
+// #define USE_CUBEMAP_TARGETS
+
+/* typedefs */
+
+typedef void(__cdecl* rasterizer_dx9_set_target_t)(e_rasterizer_target, int32, bool);
+typedef void(__cdecl* rasterizer_dx9_set_target_as_texture_t)(int16, e_rasterizer_target);
+typedef bool(__cdecl* rasterizer_dx9_set_render_target_internal_t)(IDirect3DSurface9*, IDirect3DSurface9*, bool);
+
+/* globals */
+
+rasterizer_dx9_set_target_t p_rasterizer_dx9_set_target;
+rasterizer_dx9_set_target_as_texture_t p_rasterizer_dx9_set_target_as_texture;
+rasterizer_dx9_set_render_target_internal_t p_rasterizer_dx9_set_render_target_internal;
+
+real32 g_sun_size = 0.2f;
+
+IDirect3DTexture9* g_backbuffer_texture = NULL;
```

Comment:
tried to find an argument on why these should stay, and I couldn't find anything reasonable.

Reply:
Going to remove all the ones that we're not currently using in the re-written functions.

Reply:
shouldn't be using any of these because we could already access them from the game itself.

Reply:
There's no point going back and forth between the dll and the game if we already have the functions reversed that use them.
It's extra functions rather than having the global accessed locally.

Reply:
that's why I proposed mapping the game globals inside a structure for easy access?

Reply:
At the end of the day it's a function call that isn't really needed when the global can be used.